src/api/faq/faq.model.js

```diff
@@ -1 +1,21 @@
+const mongoose = require('mongoose');
 
+const { Schema } = mongoose;
+
+const FAQSchema = new Schema({
+  question: {
+    type: String,
+    required: true,
+  },
+  answer: {
+    type: String,
+    required: true,
+  },
+  dateUpdated: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  },
+});
+
+mongoose.model('faqs', FAQSchema);
```

Comment:
Update this to:

`module.exports = mongoose.model('faqs', FAQSchema);`

Then in any other file you can directly require the module without having to import mongoose.

Reply:
Done!